client/src/app/pages/profile/profile.component.ts

```diff
@@ -1,10 +1,25 @@
 import { Component } from '@angular/core';
-
+import { ProfileService } from 'src/app/services/profile.service';
+import { Post } from 'src/app/models/post.interface';
 @Component({
   selector: 'app-profile',
   templateUrl: './profile.component.html',
   styleUrls: ['./profile.component.scss']
 })
 export class ProfileComponent {
 
+  service: ProfileService = new ProfileService();
```

Comment:
Don't initialize injectables, add them in a constructor, like this:
`constructor(private readonly profileService: ProfileService) { }`